GateMaterials.db

```diff
@@ -220,22 +366,55 @@ PTFE:   d= 2.18 g/cm3 ; n=2 ; state=solid
         +el: name=Carbon     ; n=1
         +el: name=Fluorine   ; n=2
 
-LYSO:   d=7.36 g/cm3; n=4 ; state=Solid
-        +el: name=Lutetium ; f=0.714467891
-        +el: name=Yttrium ; f=0.04033805
-        +el: name=Silicon; f=0.063714272
-        +el: name=Oxygen; f=0.181479788
+LYSO:   d=7.36 g/cm3; n=4 ; state=solid
+        +el: name=Lutetium   ; f=0.714467891
+        +el: name=Yttrium    ; f=0.04033805
+        +el: name=Silicon    ; f=0.063714272
+        +el: name=Oxygen     ; f=0.181479788
 
- LYSOold:   d=5.37 g/cm3; n=4 ; state=Solid
+ LYSOold:   d=5.37 g/cm3; n=4 ; state=solid
         +el: name=Lutetium   ; f=0.31101534
         +el: name=Yttrium    ; f=0.368765605
         +el: name=Silicon    ; f=0.083209699
         +el: name=Oxygen     ; f=0.237009356
 
+LYSO-Ce-Hilger:   d=7.10 g/cm3; n=5 ; state=solid
+        +el: name=Lutetium   ; f=0.713838658203075
+        +el: name=Yttrium    ; f=0.040302477781781
+        +el: name=Silicon    ; f=0.063721807284236
+        +el: name=Oxygen     ; f=0.181501252152072
+        +el: name=Cerium     ; f=0.000635804578835201
+
+LaBr3: d=5.29 g/cm3 ; n=2
+        +el: name=Bromine    ; n=3
+        +el: name=Lanthanum  ; n=1
+
+LaBr3_VLC: d=5.06 g/cm3 ; n=2
```

Comment:
Any clue what's VLC?

Reply:
I could not find any information about what "VLC" means in this context. But in all references that I looked at while searching the interwebs about lanthaum bromide, the density was always 5.06 or 5.08 g/cm3, I could not find any that has 5.29. So I am keeping only one LaBr3 material and I drop the VLC suffix.
https://en.wikipedia.org/wiki/Lanthanum%28III%29_bromide